source/getting-started/index.md

```diff
@@ -16,8 +17,8 @@ This tool provides:
 ### Node.js and npm
 
 Ember CLI is built with JavaScript, and expects the [Node.js](https://nodejs.org/)
-runtime. It also requires dependencies fetched via [npm](https://www.npmjs.com/). npm is packaged with Node.js, so if your computer has Node.js
-installed you are ready to go.
+runtime. It also requires dependencies fetched via [npm](https://www.npmjs.com/), the package manager for Node.js. npm is packaged with Node.js, so if your computer has Node.js
```

Comment:
Please manually break lines that are longer than 120 characters per the [Style Guide](https://github.com/emberjs/guides/blob/master/CONTRIBUTING.md#style-guide).